xiloader/xiloader.vcxproj

```diff
@@ -14,13 +14,13 @@
     <ProjectGuid>{1F160572-CDDD-4485-BBE4-AE854E36A2E6}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>xiloader</RootNamespace>
-    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
can you edit this back please? everything else is fine.

Reply:
+1

Reply:
Reverted in https://github.com/LandSandBoat/xiloader/pull/2/commits/b1be330641290e82a9992d469c09c6278a82e456